packages/component-utils/test/mocha.opts

```diff
@@ -0,0 +1,5 @@
+test/unit
+--compilers ts:@packages/ts/register,coffee:@packages/coffee/register
+--timeout 10000
+--recursive
+--require test/pretest.ts
```

Comment:
new packages should probably not use mocha.opts, since it's deprecated. you should just be able to add these arguments to the `mocha` command in `package.json` directly

Reply:
sweet! will do

Reply:
Is this really better? The line is _huge_ lol

Reply:
eh, mocha.opts is just going to be deprecated eventually. you could move it into a shell script if you're looking to clean up the package.json. plus since you don't have any .coffee files in your specs, you should be able to skip the `coffee:@packages/coffee/register` "compiler" configuration